sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -125,11 +130,14 @@ def load_model(self) -> ModelT:
     return self._unkeyed.load_model()
 
   def run_inference(
-      self, batch: Sequence[Tuple[KeyT, ExampleT]], model: ModelT,
-      **kwargs) -> Iterable[Tuple[KeyT, PredictionT]]:
+      self,
+      batch: Sequence[Tuple[KeyT, ExampleT]],
+      model: ModelT,
+      extra_kwargs: Optional[Dict[str, Any]] = None
+  ) -> Iterable[Tuple[KeyT, PredictionT]]:
     keys, unkeyed_batch = zip(*batch)
     return zip(
-        keys, self._unkeyed.run_inference(unkeyed_batch, model, **kwargs))
+        keys, self._unkeyed.run_inference(unkeyed_batch, model, extra_kwargs))
```

Comment:
These arguments should be passed through if they exist and not otherwise.

Reply:
why?

Reply:
But the fact is the interface _has_ grown, extra_kwargs is an argument on `ModelHandler.run_inference`.

Reply:
Done.

Reply:
Caught up with @ryanthompson591. We're going to stick with passing (renamed) `inference_args` for all frameworks, but for sklearn, raise an exception if a non-empty value is passed.